app/views/layouts/application.html.erb

```diff
@@ -196,6 +196,12 @@
         <%= link_to "<i class='fas fa-chevron-circle-left fa-fw'></i> Return to Single Cell Portal".html_safe, site_path %>
       <% end %>
     </div>
+    <div class="footer-social-media-icons-block pull-right ">
+    <%= link_to "<i class='fab fa-youtube'></i>".html_safe, "https://www.youtube.com/channel/UCp_UGD74MW8up1YkWZrWkKw", target: :_blank  %>
+    &nbsp;
+    <%= link_to "<i class='fab fa-twitter-square'></i>".html_safe, "https://twitter.com/SingleCellBroad", target: :_blank  %>
```

Comment:
I think our instrumentation would let us query by e.g. `fa-twitter-square`, but that's not especially intuitive or durable.  Refinements toward those ends:

```suggestion
    <%= link_to "<i class='fab fa-youtube' data-analytics-name="youtube-link"></i>".html_safe, "https://www.youtube.com/channel/UCp_UGD74MW8up1YkWZrWkKw", target: :_blank  %>
    &nbsp;
    <%= link_to "<i class='fab fa-twitter-square' data-analytics-name="twitter-link"></i>".html_safe, "https://twitter.com/SingleCellBroad", target: :_blank  %>
```